### PR TITLE
Revert "Fix Env (#18)"

### PR DESCRIPTION
**Pull request by iocedano**
This reverts commit 8569ab547f1b0128d3259ee0d88a66a56358947e.

### CODE REVIEW

**Review by adnanjt — APPROVED** [review]
LGTM this should be merged first

File: app/views/DR/Advices.js [hunk 1]
```diff
@@ -3,8 +3,8 @@ import { BackHandler, ScrollView, StyleSheet, View } from 'react-native';
 
 import iconAdvertisement from '../../assets/images/idea.jpg';
 import imgAdvertisement from '../../assets/images/recommendations.jpg';
-import HeaderImage from '../../components/DR/ActionCards/HeaderImage';
-import List from '../../components/DR/ActionCards/List';
+import HeaderImage from '../../components/HeaderImage';
```
Comment (inteljudajocu) [inline app/views/DR/Advices.js]: Update path of components, these components are still in DR/ActionCards and not in DR

Comment (inteljudajocu) [reply app/views/DR/Advices.js]: Home screen is duplicated in entry.js

Comment (adnanjt) [reply app/views/DR/Advices.js]: The home screen I fixed it somewhere else but Its not merged yet

File: app/components/DR/ButtonUpDrawer.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ export default class BottomUpPanel extends Component {
       start: height - this.props.startHeight,
       end: this.props.topEnd,
       min: this.props.topEnd,
-      animates: [() => this.animatedHeight],
+      animates: [() => this._animatedHeight],
```
Comment (D-Rosa99) [inline app/components/DR/ButtonUpDrawer.js]: why the _ ??

eslint have the entire file in red for this underscore 
otherside all is ok

image

Comment (inteljudajocu) [reply app/components/DR/ButtonUpDrawer.js]: Done

**Comment by D-Rosa99** [comment]
All the screen are duplicated

image

File: app/components/DR/ButtonUpDrawer.js [hunk 3]
```diff
@@ -127,15 +126,16 @@ export default class BottomUpPanel extends Component {
             {/* Scrollable content */}
             <ScrollView
               ref={scrollView => {
-                this.scrollViewRef = scrollView;
+                this._scrollView = scrollView;
```
Comment (D-Rosa99) [inline app/components/DR/ButtonUpDrawer.js]: what's the purpose of the underscore??

**Review by D-Rosa99 — APPROVED** [review]

**Pull request merged**